media/css/firefox/developer-quantum-whatsnew.scss

```diff
@@ -0,0 +1,181 @@
+// Intro
+.section-intro {
+    background-color: #fff;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    text-align: center;
+
+    @media #{$mq-tablet} {
+        @include background-size(1440px 1034px);
+        background-image: url('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png');
+        background-position: 0 0;
+        padding-bottom: 40px;
+
+        &:after {
+            @include background-size(3000px 62px);
+            @include section-curve;
+            background-image: url('/media/img/firefox/products/developer/quantum/curve-download-bottom-whatsnew.svg');
+            height: 62px;
+            bottom: -1px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include at2x('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png', 1780px, 1278px);
+        background-position: 0 -169px;
+        padding: 40px 0 60px;
+    }
+
+    .page-title {
+        @include font-size-level2;
+        text-transform: uppercase;
+        margin: 0 auto 20px;
+
+        &:before {
+            display: block;
+            content: '';
+            @include at2x('/media/img/firefox/products/developer/quantum/dev-edition-logo.png', 105px, 108px);
+            margin: 0 auto 20px;
+            width: 105px;
+            height: 108px;
+        }
+    }
+
+    .tagline {
+        max-width: $width-tablet;
+        text-align: center;
+        margin: 0 auto 80px;
+    }
+
+    .features-list {
+        li {
+            margin-bottom: 40px;
+
+            @media #{$mq-tablet} {
+                margin-bottom: 0;
+            }
+        }
+
+        h3 {
+            @include font-size(18px);
+            font-weight: normal;
+            text-transform: uppercase;
+
+            @media #{$mq-phone-wide} {
+                @include font-size(13px);
+            }
+
+            @media #{$mq-desktop} {
+                @include font-size(18px);
+            }
+        }
+
+        a.button.button-hollow {
+            color: $base-blue;
+            border-color: $base-blue;
+            @include font-size-level5;
+            border-radius: 100px;
+            margin-top: 20px;
+
+            @media #{$mq-phone-wide} {
+                padding: .5em 18px;
+                @include font-size(13px);
+            }
+
+            @media #{$mq-desktop} {
+                padding: .5em 36px;
+            }
+        }
+    }
+}
+
+@supports (display: grid) {
+    @media #{$mq-phone-wide} {
+        .gallery-list {
+            grid-gap: 10px 5px;
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+}
+
+.section-newsletter {
+    padding: 100px 0;
+
+    .content {
+        padding: 0;
+    }
+
+    .form-title {
+        margin-bottom: 20px;
+        text-align: center;
+
+        @media #{$mq-tablet} {
+            margin-bottom: 0;
+            text-align: right;
+        }
+
+        h3 {
+            @include font-size(18px);
```

Comment:
We're trying to avoid arbitrary font sizes and stick to a set scale (alas, not properly documented anywhere yet, but the example is at http://craigcook.github.io/etc/typescale/). 

H3 elements should be level3 by default so you may not need to declare any other sizing here but if you do, try to use one of the mixins. If you can't use a mixin, at least use sizes within the defined scale (12px, 14px, 16px, 18px, 24px, 36px, 48px, 60px, 72px). This is another place where I'm willing to disregard the mockup and impose some consistency of our own.